pipelines/install/06-smoker.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: run smoker
+  become: true
```

Comment:
The `smoker` role doesn't work with `become: true` and there is no `become: false`. That's why you see `become: true` on the roles that should `become: true` at the `roles` level, not at the top level.

Reply:
I think `become: false` exists.

Reply:
When I tried it, it didn't work. Perhaps that's been fixed since but that's the reason it's this way today.

Note that the reason it doesn't run is that chromium doesn't want to run as root. Rather than override it, I let smoker run as the normal user.